scripts/cve-reports/send-scan.py

```diff
@@ -100,13 +109,12 @@ def parse_sarif(filename):
                 "priority": severity_to_priority_map.get(severity, "Lowest"),
             }
         )
-
     return record_list
 
 
-def main(report_path, jira_url):
+def main(report_path, jira_url, gh_meta=False, verbose=False):
```

Comment:
```suggestion
def main(report_path, jira_url, gh_meta: Optional=False, verbose: Optional=False):
```

Reply:
I wonder if we need to set the default to `False`, if the argparse is already setting them to False.